.clang-tidy

```diff
@@ -1,5 +1,8 @@
+HeaderFilterRegex: '((^(?!\/share\/openPMD\/).*)*include\/openPMD\/.+\.hpp|src\/^(?!binding).+\.cpp$)'
```

Comment:
Hm, do we filter too many files here?
My IDE shows more locations than the examples...